debugger/src/debug.c

```diff
@@ -1048,12 +1068,24 @@ void debug_init(void)
 		    NULL);
 	grantpt(pty_master);
 	unlockpt(pty_master);
+#if GTK_CHECK_VERSION(3, 0, 0)
+	pty = vte_pty_new_foreign_sync(pty_master, NULL, NULL);
+	vte_terminal_set_pty(VTE_TERMINAL(terminal), pty);
+	g_object_unref(pty);
+	scrollbar = gtk_scrollbar_new(GTK_ORIENTATION_VERTICAL, gtk_scrollable_get_vadjustment(GTK_SCROLLABLE(terminal)));
+	gtk_widget_set_can_focus(GTK_WIDGET(scrollbar), FALSE);
+#else
 	vte_terminal_set_pty(VTE_TERMINAL(terminal), pty_master);
 	scrollbar = gtk_vscrollbar_new(GTK_ADJUSTMENT(VTE_TERMINAL(terminal)->adjustment));
 	GTK_WIDGET_UNSET_FLAGS(scrollbar, GTK_CAN_FOCUS);
```

Comment:
This line could be using the GTK3 version

Reply:
If you're talking about the use of `gtk_widget_set_can_focus()`, that's right and it's fixed.
The remaining code about PTY initialization seemingly has to be different though.

Reply:
Yes I'm talking about `gtk_widget_set_can_focus()` I don't really know about the PTY stuff, but it seem that it has indeed to be different with VTE2.91.